libs/metrics/test_positivity.py

```diff
@@ -155,10 +227,74 @@ def calculate(self, dataset: MultiRegionDataset, diff_days: int) -> MultiRegionD
         wide_date_df = df.loc[self._column, :]
         # Optional optimization: The following likely adds the variable/field/column name back in
         # to the index which was just taken out. Consider skipping reindexing.
-        return _make_output_dataset(dataset, self._name, wide_date_df, CommonFields.TEST_POSITIVITY)
+
+        return self._filter_recent(
+            _make_output_dataset(dataset, self._name, wide_date_df, CommonFields.TEST_POSITIVITY),
+            most_recent_date,
+        )
+
+
+class OldMethod(Method):
+    """A method of calculating test positivity using smoothed POSITIVE_TEST and NEGATIVE_TESTS."""
+
+    @property
+    def name(self) -> timeseries.DatasetName:
+        return timeseries.DatasetName("OldMethod")
```

Comment:
hmm if this is used elsewhere I would lean towards changing it.  hard to know what it is without context

Reply:
It shows up in the provenance. Renamed.